### ISSUE TITLE
Can't deposit Elements items into PPRD (Elements upgrade has finished)

### ISSUE THREAD

**Issue by mello99** [VTUL/vtechworks#236]
Hi, all, I can't deposit Elements items into PPRD. This issue has occurred since early this afternoon after the Elements upgrade had been completed on ed.lib.vt.edu. Whenever I try to deposit Elements items into PPRD, I get the following message (see below). I asked Keith B. if he had any idea what was going on and he said no. 

Keith G. - do you think it would be wise to revert to an earlier version of my XSLT file, or do you think that something else could be causing this blockage? Thanks so much, hope all is well.

<img width="935" alt="screen shot 2016-04-19 at 2 33 50 pm" src="https://cloud.githubusercontent.com/assets/5408616/14655229/59b0f978-064f-11e6-8189-96f997dab6be.png">

**Comment by keithgee** [comment]
@mello99, I found error messages in the DSpace log file. Here's the abbreviated version:

```
ERROR uk.co.symplectic.elements.servlets.RepositoryServlet @ Caught ElementsException
uk.co.symplectic.elements.ElementsException: uk.co.symplectic.repository.dspace.DSpaceElementsException: Unable to write metadata update
        ... 26 more
Caused by: java.sql.SQLException:Invalid metadata field: [dc.rights[*]]
```

I reverted commits 2df8a111 and 2009b5c758, which added dc.rights[*| stuff to the XSLT file, and I was able to do a deposit from Elements to the repository.

Are you able to deposit now as well?

**Comment by mello99** [comment]
Hi, @keithgee , thanks so much for making that change. Yes, I was able to successfully deposit an Elements item into VTW!

Here's a list of what went **right**:

1) The full CC rights statement, in this case "Attribution-NonCommercial-NoDerivatives 4.0 International", went into "dc.rights"
2) The URI for the corresponding rights statement - "http://creativecommons.org/licenses/by-nc-nd/4.0/" - went into "dc.rights.uri".

Having this information in both of these fields - "dc.rights" AND "dc.rights.uri" - allowed DSpace to successfully generate a CC file - YAY!!!

3) The word "chapter(s)" appeared in the "dc.format.extent" field, which is where chapter information is mapped to

Here's a list of what went **wrong**, which needs to be addressed on our end:

1) The "dc.relation.ispartof" field is still duplicated
2) The string "page(s)" was not appended to the "dc.format.extent" field. I added the "page(s)" string to provide greater context to this field; perhaps I can add it back (I'm not sure if the changes made to this field were reverted in this instance of the XSLT, so I'll check again).

It's looking much better - progress is being made 👍
